libs/ngx-dsfr/radio/src/lib/radio.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="fr-form-group">
+  <fieldset class="fr-fieldset"
+            [ngClass]="{'fr-fieldset--inline': inline, 'fr-fieldset--error': hasFailed, 'fr-fieldset--valid': hasSucceeded}"
+            [attr.aria-labelledby]="hasFailed ? 'radio-error-legend radio-error-desc-error' : hasSucceeded ? 'radio-valid-legend radio-valid-desc-valid' : null"
+            [attr.role]="hasFailed || hasSucceeded ? 'group' : null"
+            [disabled]="disabled">
+    <legend class="fr-fieldset__legend fr-text--regular">
+      {{legend}}
+      <span *ngIf="hint" class="fr-hint-text">{{hint}}</span>
+    </legend>
+    <div class="fr-fieldset__content">
+      <div *ngFor="let item of items" class="fr-radio-group" [ngClass]="'fr-radio-group--' + size">
+        <input [(ngModel)]="value" (change)="valueChanged(item.value)" type="radio" [id]="item.id" [name]="name"
```

Comment:
First, it seems to me you can use directly `onChange` rather than having another method like `valueChanged` which only calls `onChange` internally.
Second, it took me a while to understand how it could work ( which tells me it's not easily readable and, since, we're doing this open source so other people can use it, readability is important in this project ) but I think the `[(ngModel)]` part works because it relies on [`RadioControlValueAccessor`](https://angular.io/api/forms/RadioControlValueAccessor). I'd suggest to replace it by this :
```html
<input [checked]="item.value === controlValue"
```
where `controlValue`, or `radioGroupValue`, would be your current `value` if I understand correctly...
Just remember that this Component represents the radio group and not just an individual radio button.
As a radio group, this Component has a value that changes during its lifetime ( when users are clicking on radio buttons ) but each `<input>` has a value that is set once and it's its `checked` property that changes...